openupgradelib/openupgrade.py

```diff
@@ -1668,3 +1669,26 @@ def rename_property(cr, model, old_name, new_name):
     cr.execute(
         "update ir_property set name=%s where fields_id in %s",
         (new_name, field_ids))
+
+def delete_template_translations(cr, module, model, xml_ids):
```

Comment:
Prefer `delete_record_translations`

Reply:
The model argument is void. You can take it from ir_model_data.

Reply:
Done.